src/ansys/dpf/post/fluid_simulation.py

```diff
@@ -177,6 +178,15 @@ def mesh_info(self) -> FluidMeshInfo:
             self._mesh_info = FluidMeshInfo(self._model.metadata.mesh_info)
         return self._mesh_info
 
+    def zone_mesh(self, zone_id: int) -> Mesh:
```

Comment:
@PProfizi can we accept a zone name as well please? This would make it more consistent with the requests on phases